### PR TITLE
Installation Shell Script  changes

### PR DESCRIPTION
**Pull request by kiran-aot**
Modified the shell script with suggestions

### CODE REVIEW

**Review by alanraju-aot — DISMISSED** [review]

File: scripts/install.bash [hunk 1]
```diff
@@ -1,5 +1,5 @@
 #!/bin/bash
-ipadd=$(hostname -i)
+ipadd=$(ip addr show eth0 | grep "inet\b" | awk '{print $2}' | cut -d/ -f1)
```
Comment (shibin-aot) [inline scripts/install.bash:2]: Is this tested? test this in few machines, we had some issues with the ip command earlier.
Also, you can place this on a global variable as this doesn't change during execution.

File: scripts/install.bash [hunk 2]
```diff
@@ -122,13 +113,11 @@ function forms-flow-analytics
 
 function forms-flow-bpm
 {
-    cd ..
-    ipadd=$(hostname -i)
-
+    ipadd=$(ip addr show eth0 | grep "inet\b" | awk '{print $2}' | cut -d/ -f1)
```
Comment (shibin-aot) [inline scripts/install.bash:116]: Lot of duplication of this code, can be moved into global field

File: docker-compose/docker-compose-local.yml [hunk 3]
```diff
@@ -26,6 +26,9 @@ services:
       - KEYCLOAK_PASSWORD=${KEYCLOAK_ADMIN_PASSWORD:-changeme}
     ports:
       - "8080:8080"
+    depends_on:
```
Comment (shibin-aot) [inline docker-compose/docker-compose-local.yml:29]: Doesn't need to mentioned this, keycloak-db doesn't take time to come up and since the service is placed above keycloak, keycloak-db always comes up before keycloak

**Review by shibin-aot — APPROVED** [review]

File: scripts/install.bash (hunk 1 above)
Comment (kiran-aot) [reply scripts/install.bash:2]: Tested in in ubuntu and centos linux distros , let me test in possible other distros

File: scripts/install.bash (hunk 2 above)
Comment (kiran-aot) [reply scripts/install.bash:116]: noted and modify it

**Review by shibin-aot — DISMISSED** [review]

**Review by alanraju-aot — DISMISSED** [review]

File: scripts/install.bash [hunk 4]
```diff
@@ -1,5 +1,5 @@
 #!/bin/bash
-ipadd=$(hostname -i)
+ipadd=$(hostname -I | awk '{print $1}')
```
Comment (sumesh-aot) [inline scripts/install.bash:2]: Can't we refer to the container names directly than IP address if we add all containers to the same network ?

File: docker-compose/docker-compose-local.yml [hunk 5]
```diff
@@ -26,9 +26,13 @@ services:
       - KEYCLOAK_PASSWORD=${KEYCLOAK_ADMIN_PASSWORD:-changeme}
     ports:
       - "8080:8080"
+    links:
```
Comment (abhilash-aot) [inline docker-compose/docker-compose-local.yml:29]: why is this file changed to -local.yml .

Comment (alanraju-aot) [reply docker-compose/docker-compose-local.yml:29]: > why is this file changed to -local.yml .

@shibin-aot

**Review by alanraju-aot — APPROVED** [review]

**Review by abilpraju-aot — APPROVED** [review]
